### ISSUE TITLE
Problem and suggestion

### ISSUE THREAD

**Issue by lilbeefbroth** [lucasteles/InputDisplay#8]
1.  It looks like in the color menu, turning Alpha to 0 will make the background transparent which is crucial to OBS and streaming in general......but it's not working for me.  I can't get the background transparent at all.  
2.  PLEASE PLEASE PLEASE make an update that somehow can output the settings to a local file to be used as a browser source or something....having to keep the window open and on screen can be a real hassle when trying to stream a game on one screen and OBS on another.

thank you

**Comment by lucasteles** [comment]
1. The background alpha color does not work; you should set a transparent color in OBS, for the window. Here is a guide: https://github.com/lucasteles/InputDisplay/blob/master/how_to_obs.md

2. There is a file called `settings.json`, which has all your settings, you can save and/or replace it for multiples configurations for now

**Comment by lilbeefbroth** [comment]
Yep that’s what I ended up doing shortly after sending the message.  ThanksOn Jun 2, 2025, at 1:12 PM, Lucas Teles ***@***.***> wrote:﻿lucasteles left a comment (lucasteles/InputDisplay#8)
The background alpha color does not work, you should set a transparent color in OBS, for the window, here a guide: https://github.com/lucasteles/InputDisplay/blob/master/how_to_obs.md
